doc/source/overview.rst

```diff
@@ -85,36 +88,44 @@ The first stop for pandas issues and ideas is the `Github Issue Tracker
-inquiries for Lambda Foundry should be sent to: support@lambdafoundry.com
+Community
+---------
 
-Credits
--------
+pandas is actively supported today by a community of like-minded individuals around 
+the world who contribute their valuable time and energy to help make open source 
+pandas possible. Thanks to `all of our contributors <https://github.com/pandas-dev/pandas/graphs/contributors>`__.
+
+If you're interested in contributing, please
+visit `Contributing to pandas webpage <http://pandas.pydata.org/pandas-docs/stable/contributing.html>`__.
+
+pandas is a `NUMFocus <https://www.numfocus.org/open-source-projects/>`__ sponsored project.
+This will help ensure the success of development of pandas as a world-class open-source
+project, and makes it possible to `donate <https://pandas.pydata.org/donate.html>`__ to the project.
 
-pandas development began at `AQR Capital Management <http://www.aqr.com>`__ in
-April 2008. It was open-sourced at the end of 2009. AQR continued to provide
-resources for development through the end of 2011, and continues to contribute
-bug reports today.
+Project Governance
+------------------
 
-Since January 2012, `Lambda Foundry <http://www.lambdafoundry.com>`__, has
-been providing development resources, as well as commercial support,
-training, and consulting for pandas.
+The governance process that pandas project has used informally since its inception in 2008 is formalized in `Project Governance documents <https://github.com/pandas-dev/pandas-governance>`__ .
+The documents clarify how decisions are made and how the various elements of our community interact, including the relationship between open source collaborative development and work that may be funded by for-profit or non-profit entities.
 
-pandas is only made possible by a group of people around the world like you
-who have contributed new code, bug reports, fixes, comments and ideas. A
-complete list can be found `on Github <http://www.github.com/pandas-dev/pandas/contributors>`__.
+Wes McKinney is the Benevolent Dictator for Life (BDFL).
 
 Development Team
-----------------
+-----------------
 
-pandas is a part of the PyData project. The PyData Development Team is a
-collection of developers focused on the improvement of Python's data
-libraries. The core team that coordinates development can be found on `Github
-<http://github.com/pydata>`__. If you're interested in contributing, please
-visit the `project website <http://pandas.pydata.org>`__.
+The list of the  Core Team members and more detailed information can be found on the `people’s page <https://github.com/pandas-dev/pandas-governance/blob/master/people.md>`__ of the governance repo.
+ 
+
+Institutional Partners
+----------------------
+
+Institutional Partners are companies and universities that support the project by employing contributors. Current Institutional Partners include:
+
+* `Continuum Analytics <https://www.anaconda.com/>`__
```

Comment:
Continuum just changed their name to Anaconda, could you update that (URL is the same)?

Reply:
I would maybe link here to the website page instead of listing them? 
Just to limit the places where we need to update things ...